libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/pages/HeaderFooterPage.kt

```diff
@@ -94,17 +96,19 @@ fun HeaderFooterPage(
                         .run {
                             if (isScrollable) {
                                 verticalScroll(rememberScrollState())
+                                    // Make sure the scrollable content takes the full available height
+                                    .height(IntrinsicSize.Max)
                             } else {
                                 Modifier
                             }
                         }
                         // Apply insets here so if the content is scrollable it can get below the top app bar if needed
                         .padding(contentInsetsPadding)
-                        .weight(1f),
+                        .weight(1f, fill = true),
                 ) {
                     // Header
                     header()
-                    Box(modifier = Modifier.weight(1f)) {
+                    Box {
                         content()
```

Comment:
Maybe we should add some padding between the footer and the content? This looks a bit weird without

Reply:
Hmm... maybe, but maybe that should be added for the content on each screen? Otherwise we need to remember we have `10-16.dp` here and substract it from whatever we have in the designs.

Reply:
@ganfra just to remind you of this question: should we add a default padding for all screens and then diff with the designs or leave it at 0 and just apply the padding to each content component so it matches the designs 1:1?